TeamCode/src/main/java/org/firstinspires/ftc/teamcode/utils/RobotConfig.kt

```diff
@@ -0,0 +1,39 @@
+@file:Suppress("ktlint:standard:property-naming")
+
+package org.firstinspires.ftc.teamcode.utils
+
+import org.firstinspires.ftc.teamcode.utils.RobotConfig.debug
+
+/**
+ * This is an immutable object representing robot configuration.
+ *
+ * It is meant to orchestrate FTC Dashboard and other configuration together.
+ * Certain sub-objects are annotated with `@Config`. This designates them as FTC Dashboard
+ * configuration that can be modified at runtime. **To permanently change these values, you must
+ * also modify the code!** The configuration can also change during initialization depending on
+ * various build constants like [debug].
```

Comment:
```suggestion
 * It is meant to orchestrate FTC Dashboard and other configuration together. Certain sub-objects
 * are annotated with `@Config`. This designates them as FTC Dashboard configuration that can be
 * modified at runtime. **To permanently change these values, you must also modify the code!** The
 * configuration can also change during initialization depending on various build constants like
 * [debug].
```

This is a nit, so its not blocking the PR from being merged.

This change reorganizes the comment to be 100 characters wide, which is what I've been using in the past. Unfortunately `ktlint` doesn't seem to support enforcing comment width, so this is something I manually look for.